spec/forms/support_interface/science_gcse_form_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe SupportInterface::ScienceGcseForm do
+  let(:qualification) { create(:application_qualification) }
+
+  subject(:form) { described_class.build_from_qualification(qualification) }
+
+  describe 'validations' do
+    it { is_expected.to validate_length_of(:grade).is_at_most(256) }
```

Comment:
Should we be testing that the validation is at most  `ApplicationQualification::MAX_QUALIFICATION_GRADE_LENGTH`?